conf/application.conf

```diff
@@ -352,6 +352,17 @@ feature {
     maxTps = 10
     timeoutInSec = 5
   }
+
+  banner {
+    home {
+      enabled = true
+      heading.en = "Help improve GOV.UK"
```

Comment:
Can we put heading text in Message or any specific reason ?

Reply:
If you put the text in message then it is part of the codebase which I don't want. The text should be configurable at runtime so via app-config-*.
Which means no code change, no  lengthy QA check to push a new banner except a design check is staging. It speeds up massively the change of the banner.

Reply:
understood , great